WooCommerce/Classes/ViewRelated/Dashboard/Settings/SettingsViewController.swift

```diff
@@ -223,6 +223,12 @@ private extension SettingsViewController {
         cell.textLabel?.text = NSLocalizedString("Help & Support", comment: "Contact Support Action")
     }
 
+    func configureCardReaders(cell: BasicTableViewCell) {
+        cell.accessoryType = .disclosureIndicator
+        cell.selectionStyle = .default
+        cell.textLabel?.text = NSLocalizedString("Manage card readers", comment: "Manage card readers Action")
```

Comment:
The comment passed to NSLocalizedString is meant to help translators figure out the context for that screen. Would it make sense for this to read "Navigates to List of Card Readers screen" (or similar, but better worded)?

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/3624/commits/231ca271cc5b2a580c63c90ff89f9d6f5ad90fa3